source/Process.Application/Transactions/WholesaleServices/WholesaleServicesRequestAcceptedMapper.cs

```
@@ -123,19 +124,19 @@ private static MeasurementUnit MapMeasurementUnit(WholesaleServicesRequestSeries
         {
             WholesaleServicesRequestSeries.Types.QuantityUnit.Kwh => MeasurementUnit.Kwh,
             WholesaleServicesRequestSeries.Types.QuantityUnit.Pieces => MeasurementUnit.Pieces,
-            WholesaleServicesRequestSeries.Types.QuantityUnit.Unspecified => throw new InvalidOperationException("Could not map quantity unit"),
+            WholesaleServicesRequestSeries.Types.QuantityUnit.Unspecified => MeasurementUnit.Kwh,
```

Is this correct? AFAIK the `Unspecified` enum value in protobuf should never be used, thus we should throw an exception if we receive one. It is not intended for a default value or something like that (https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum)